test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/PageWithNamespace_DesignTime.codegen.cs

```diff
@@ -1,3 +1,5 @@
+// <auto-generated/>
```

Comment:
I've updated this to only do this in design time. We don't add pragma checksum in design time.